### PR TITLE
Fix some documentation in ./src/diffusers/models/adapter.py

### PR DESCRIPTION
**Pull request by ahnjj**
# What does this PR do?
Fixes https://github.com/huggingface/diffusers/issues/9567

Fixes # (issue)
- Unified two expressions with the same meaning into one. (arguments / parameters -> args)
- Concise Summary and argument descriptions
- Corrected Grammar

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [x] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

This PR tries to attempt a solution at one of the submodules listed in https://github.com/huggingface/diffusers/issues/9567 so I think @a-r-r-o-w is the best to review it. Alongside the same, @charchit7 @yijun-lee and @SubhasmitaSw were also working on the same, so just a ping for the update on the same.

### CODE REVIEW

File: src/diffusers/models/adapter.py [hunk 1]
```diff
@@ -145,50 +148,47 @@ def save_pretrained(
...
                 A map that specifies where each submodule should go. It doesn't need to be refined to each
                 parameter/buffer name, once a given module name is inside, every submodule of it will be sent to the
                 same device.
 
                 To have Accelerate compute the most optimized `device_map` automatically, set `device_map="auto"`. For
                 more information about each option see [designing a device
                 map](https://hf.co/docs/accelerate/main/en/usage_guides/big_modeling#designing-a-device-map).
-            max_memory (`Dict`, *optional*):
-                A dictionary device identifier to maximum memory. Will default to the maximum memory available for each
+            max_memory (`Dict`, optional):
```
Comment (ahnjj) [inline src/diffusers/models/adapter.py:177]:
```suggestion
            max_memory (`Dict`, *optional*):
```

File: src/diffusers/models/adapter.py [hunk 2]
```diff
@@ -145,50 +148,47 @@ def save_pretrained(
...
                 [`~diffusers.models.adapter.MultiAdapter.save_pretrained`], e.g., `./my_model_directory/adapter`.
-            torch_dtype (`str` or `torch.dtype`, *optional*):
+            torch_dtype (`str` or `torch.dtype`, optional):
                 Override the default `torch.dtype` and load the model under this dtype. If `"auto"` is passed the dtype
                 will be automatically derived from the model's weights.
-            output_loading_info(`bool`, *optional*, defaults to `False`):
+            output_loading_info(`bool`, optional, defaults=False):
                 Whether or not to also return a dictionary containing missing keys, unexpected keys and error messages.
-            device_map (`str` or `Dict[str, Union[int, str, torch.device]]`, *optional*):
+            device_map (`str` or `Dict[str, Union[int, str, torch.device]]`, optional):
```
Comment (ahnjj) [inline src/diffusers/models/adapter.py:169]:
```suggestion
            device_map (`str` or `Dict[str, Union[int, str, torch.device]]`, *optional*):
```

File: src/diffusers/models/adapter.py [hunk 3]
```diff
@@ -145,50 +148,47 @@ def save_pretrained(
...
+            *Weights from XXX not initialized from pretrained model* means that the weights of XXX are not pretrained with the rest of the model. It is up to you to train those weights with a downstream fine-tuning. 
+            *Weights from XXX not used in YYY* means that the layer XXX is not used by YYY, so those weights are discarded.
 
-        Parameters:
+        Args:
             pretrained_model_path (`os.PathLike`):
                 A path to a *directory* containing model weights saved using
                 [`~diffusers.models.adapter.MultiAdapter.save_pretrained`], e.g., `./my_model_directory/adapter`.
-            torch_dtype (`str` or `torch.dtype`, *optional*):
+            torch_dtype (`str` or `torch.dtype`, optional):
```
Comment (ahnjj) [inline src/diffusers/models/adapter.py:164]:
```suggestion
            torch_dtype (`str` or `torch.dtype`, *optional*):
```

File: src/diffusers/models/adapter.py [hunk 4]
```diff
@@ -145,50 +148,47 @@ def save_pretrained(
...
                 Override the default `torch.dtype` and load the model under this dtype. If `"auto"` is passed the dtype
                 will be automatically derived from the model's weights.
-            output_loading_info(`bool`, *optional*, defaults to `False`):
+            output_loading_info(`bool`, optional, defaults=False):
```
Comment (ahnjj) [inline src/diffusers/models/adapter.py:167]:
```suggestion
            output_loading_info(`bool`, *optional*, defaults to `False`):
```

File: src/diffusers/models/adapter.py [hunk 5]
```diff
@@ -109,24 +112,24 @@ def save_pretrained(
...
-            safe_serialization (`bool`, *optional*, defaults to `True`):
-                Whether to save the model using `safetensors` or the traditional PyTorch way (that uses `pickle`).
-            variant (`str`, *optional*):
-                If specified, weights are saved in the format pytorch_model.<variant>.bin.
+                Function used to save the state dictionary. Useful on distributed training (e.g., TPUs) when one
+                need to replace `torch.save` with another method. Can also be configured using`DIFFUSERS_SAVE_MODE` environment variable.
+            safe_serialization (`bool`, optional, defaults=True):
+                If `True`, save the model using `safetensors`.
+                If `False`, save the model using the traditional PyTorch way (using `pickle`).
+            variant (`str`, optional):
```
Comment (ahnjj) [inline src/diffusers/models/adapter.py:131]:
```suggestion
            variant (`str`, *optional*):
```

File: src/diffusers/models/adapter.py [hunk 6]
```diff
@@ -145,50 +148,47 @@ def save_pretrained(
...
                 To have Accelerate compute the most optimized `device_map` automatically, set `device_map="auto"`. For
                 more information about each option see [designing a device
                 map](https://hf.co/docs/accelerate/main/en/usage_guides/big_modeling#designing-a-device-map).
-            max_memory (`Dict`, *optional*):
-                A dictionary device identifier to maximum memory. Will default to the maximum memory available for each
+            max_memory (`Dict`, optional):
+                A dictionary mapping device identifiers to their maximum memory. Default to the maximum memory available for each
                 GPU and the available CPU RAM if unset.
-            low_cpu_mem_usage (`bool`, *optional*, defaults to `True` if torch version >= 1.9.0 else `False`):
+            low_cpu_mem_usage (`bool`, optional, defaults=True if torch version >= 1.9.0 else `False`):
```
Comment (ahnjj) [inline src/diffusers/models/adapter.py:180]:
```suggestion
            low_cpu_mem_usage (`bool`, *optional*, defaults to `True` if torch version >= 1.9.0 else `False`):
```

File: src/diffusers/models/adapter.py [hunk 7]
```diff
@@ -145,50 +148,47 @@ def save_pretrained(
...
                 Speed up model loading by not initializing the weights and only loading the pre-trained weights. This
                 also tries to not use more than 1x model size in CPU memory (including peak memory) while loading the
                 model. This is only supported when torch version >= 1.9.0. If you are using an older version of torch,
                 setting this argument to `True` will raise an error.
-            variant (`str`, *optional*):
-                If specified load weights from `variant` filename, *e.g.* pytorch_model.<variant>.bin. `variant` is
+            variant (`str`, optional):
```
Comment (ahnjj) [inline src/diffusers/models/adapter.py:185]:
```suggestion
            variant (`str`, *optional*):
```

File: src/diffusers/models/adapter.py [hunk 8]
```diff
@@ -223,22 +223,20 @@ class T2IAdapter(ModelMixin, ConfigMixin):
...
-            Number of channels of Aapter's input(*control image*). Set this parameter to 1 if you're using gray scale
-            image as *control image*.
-        channels (`List[int]`, *optional*, defaults to `(320, 640, 1280, 1280)`):
-            The number of channel of each downsample block's output hidden state. The `len(block_out_channels)` will
-            also determine the number of downsample blocks in the Adapter.
-        num_res_blocks (`int`, *optional*, defaults to 2):
+    This model inherits from [`ModelMixin`]. Check the superclass documentation for the common methods, such as downloading or saving.
+
+    Args:
+        in_channels (`int`, optional, defaults=3):
```
Comment (ahnjj) [inline src/diffusers/models/adapter.py:229]:
```suggestion
        in_channels (`int`, *optional*, defaults to `3`):
```

File: src/diffusers/models/adapter.py [hunk 9]
```diff
@@ -145,50 +148,47 @@ def save_pretrained(
...
-            variant (`str`, *optional*):
-                If specified load weights from `variant` filename, *e.g.* pytorch_model.<variant>.bin. `variant` is
+            variant (`str`, optional):
+                If specified, load weights from a `variant` file (*e.g.* pytorch_model.<variant>.bin). `variant` will be
                 ignored when using `from_flax`.
-            use_safetensors (`bool`, *optional*, defaults to `None`):
-                If set to `None`, the `safetensors` weights will be downloaded if they're available **and** if the
-                `safetensors` library is installed. If set to `True`, the model will be forcibly loaded from
-                `safetensors` weights. If set to `False`, loading will *not* use `safetensors`.
+            use_safetensors (`bool`, optional, defaults=None):
```
Comment (ahnjj) [inline src/diffusers/models/adapter.py:188]:
```suggestion
            use_safetensors (`bool`, *optional*, defaults to `None`):
```

File: src/diffusers/models/adapter.py [hunk 10]
```diff
@@ -223,22 +223,20 @@ class T2IAdapter(ModelMixin, ConfigMixin):
...
+            Number of channel of each downsample block's output hidden state. The `len(block_out_channels)` determines
+            the number of downsample blocks in the Adapter.
+        num_res_blocks (`int`, *optional*, defaults to `2`):
             Number of ResNet blocks in each downsample block.
-        downscale_factor (`int`, *optional*, defaults to 8):
+        downscale_factor (`int`, *optional*, defaults to `8`):
             A factor that determines the total downscale factor of the Adapter.
         adapter_type (`str`, *optional*, defaults to `full_adapter`):
-            The type of Adapter to use. Choose either `full_adapter` or `full_adapter_xl` or `light_adapter`.
+            Type of Adapter to use. Choose either `full_adapter` or `full_adapter_xl` or `light_adapter`.
```
Comment (stevhliu) [inline src/diffusers/models/adapter.py:239]:
```suggestion
            Adapter type (`full_adapter` or `full_adapter_xl` or `light_adapter`) to use.
```

File: src/diffusers/models/adapter.py [hunk 11]
```diff
@@ -223,22 +223,20 @@ class T2IAdapter(ModelMixin, ConfigMixin):
...
-            image as *control image*.
+    Args:
+        in_channels (`int`, *optional*, defaults to `3`):
+            Number of channels of Aapter's input(*control image*). Set to 1 if you're using gray scale image.
         channels (`List[int]`, *optional*, defaults to `(320, 640, 1280, 1280)`):
-            The number of channel of each downsample block's output hidden state. The `len(block_out_channels)` will
-            also determine the number of downsample blocks in the Adapter.
-        num_res_blocks (`int`, *optional*, defaults to 2):
+            Number of channel of each downsample block's output hidden state. The `len(block_out_channels)` determines
+            the number of downsample blocks in the Adapter.
```
Comment (stevhliu) [inline src/diffusers/models/adapter.py:233]:
```suggestion
            The number of channels in each downsample block's output hidden state. The `len(block_out_channels)` determines
            the number of downsample blocks in the adapter.
```

File: src/diffusers/models/adapter.py [hunk 12]
```diff
@@ -223,22 +223,20 @@ class T2IAdapter(ModelMixin, ConfigMixin):
...
-    implements for all the model (such as downloading or saving, etc.)
+    This model inherits from [`ModelMixin`]. Check the superclass documentation for the common methods, such as downloading or saving.
 
-    Parameters:
-        in_channels (`int`, *optional*, defaults to 3):
-            Number of channels of Aapter's input(*control image*). Set this parameter to 1 if you're using gray scale
-            image as *control image*.
+    Args:
+        in_channels (`int`, *optional*, defaults to `3`):
+            Number of channels of Aapter's input(*control image*). Set to 1 if you're using gray scale image.
```
Comment (stevhliu) [inline src/diffusers/models/adapter.py:230]:
```suggestion
            The number of channels in the adapter's input (*control image*). Set it to 1 if you're using a gray scale image.
```

File: src/diffusers/models/adapter.py [hunk 13]
```diff
@@ -109,24 +112,24 @@ def save_pretrained(
...
-                The function to use to save the state dictionary. Useful on distributed training like TPUs when one
-                need to replace `torch.save` by another method. Can be configured with the environment variable
-                `DIFFUSERS_SAVE_MODE`.
-            safe_serialization (`bool`, *optional*, defaults to `True`):
-                Whether to save the model using `safetensors` or the traditional PyTorch way (that uses `pickle`).
+                Function used to save the state dictionary. Useful on distributed training (e.g., TPUs) when one
+                need to replace `torch.save` with another method. Can also be configured using`DIFFUSERS_SAVE_MODE` environment variable.
+            safe_serialization (`bool`, optional, defaults=True):
+                If `True`, save the model using `safetensors`.
+                If `False`, save the model using the traditional PyTorch way (using `pickle`).
```
Comment (stevhliu) [inline src/diffusers/models/adapter.py:130]:
```suggestion
                If `False`, save the model with `pickle`.
```

File: src/diffusers/models/adapter.py [hunk 14]
```diff
@@ -109,24 +112,24 @@ def save_pretrained(
...
+                Useful when in distributed training (e.g., TPUs) and need to call this function on all processes. 
+                In this case, set `is_main_process=True` only for the main process to avoid race conditions.
             save_function (`Callable`):
-                The function to use to save the state dictionary. Useful on distributed training like TPUs when one
-                need to replace `torch.save` by another method. Can be configured with the environment variable
-                `DIFFUSERS_SAVE_MODE`.
-            safe_serialization (`bool`, *optional*, defaults to `True`):
-                Whether to save the model using `safetensors` or the traditional PyTorch way (that uses `pickle`).
+                Function used to save the state dictionary. Useful on distributed training (e.g., TPUs) when one
+                need to replace `torch.save` with another method. Can also be configured using`DIFFUSERS_SAVE_MODE` environment variable.
```
Comment (stevhliu) [inline src/diffusers/models/adapter.py:127]:
```suggestion
                Function used to save the state dictionary. Useful for distributed training (e.g., TPUs) to replace `torch.save` with another method. Can also be configured using`DIFFUSERS_SAVE_MODE` environment variable.
```

File: src/diffusers/models/adapter.py [hunk 15]
```diff
@@ -109,24 +112,24 @@ def save_pretrained(
...
             save_directory (`str` or `os.PathLike`):
-                Directory to which to save. Will be created if it doesn't exist.
-            is_main_process (`bool`, *optional*, defaults to `True`):
-                Whether the process calling this is the main process or not. Useful when in distributed training like
-                TPUs and need to call this function on all processes. In this case, set `is_main_process=True` only on
-                the main process to avoid race conditions.
+                The directory where the model will be saved. If the directory does not exist, it will be created.
+            is_main_process (`bool`, optional, defaults=True):
+                Indicates whether current process is the main process or not. 
+                Useful when in distributed training (e.g., TPUs) and need to call this function on all processes.
```
Comment (stevhliu) [inline src/diffusers/models/adapter.py:123]:
```suggestion
                Useful for distributed training (e.g., TPUs) and need to call this function on all processes. 
```

File: src/diffusers/models/adapter.py [hunk 16]
```diff
@@ -77,11 +77,14 @@ def forward(self, xs: torch.Tensor, adapter_weights: Optional[List[float]] = Non
...
             xs (`torch.Tensor`):
-                (batch, channel, height, width) input images for multiple adapter models concated along dimension 1,
-                `channel` should equal to `num_adapter` * "number of channel of image".
+                A tensor of shape (batch, channel, height, width) representing input images for multiple adapter models, 
+                concatenated along dimension 1(channel dimension).
+                The `channel` dimension should be equal to `num_adapter` * number of channel per image.
+            
             adapter_weights (`List[float]`, *optional*, defaults to None):
-                List of floats representing the weight which will be multiply to each adapter's output before adding
+                A list of floats representing the weight which will be multiplied to each adapter's output before summing
```
Comment (stevhliu) [inline src/diffusers/models/adapter.py:85]:
```suggestion
                A list of floats representing the weights which will be multiplied by each adapter's output before summing
```

**Review by stevhliu — APPROVED** [review]
Thanks for improving, running `make fix style` should fix the failing CI tests!

**Comment by a-r-r-o-w** [comment]
Hey, seeing some quality tests failing. Could you run `make style` and push the changes?

**Comment by ahnjj** [comment]
> Hey, seeing some quality tests failing. Could you run `make style` and push the changes?

Thank you for your help! I run make style and made a new commit! 
Should I wait til this is merged ? or need extra adjustment?

**Comment by a-r-r-o-w** [comment]
It seems like you're on a different version of `ruff` than what we use on our CI, causing unrelated changes. Could you make sure your ruff version is `0.1.5`, run `make style` again and push the changes? Sorry about the inconvenience!

**Comment by ahnjj** [comment]
> It seems like you're on a different version of `ruff` than what we use on our CI, causing unrelated changes. Could you make sure your ruff version is `0.1.5`, run `make style` again and push the changes? Sorry about the inconvenience!

Thanks for your help!
I installed 0.1.5 version ruff, run make style and it shows an error with 'doc-builder'. 
Do you happen to know regarding this? Thank you so much!
```
make: doc-builder: Command not found
make: *** [Makefile:59: style] Error 127
```

**Comment by a-r-r-o-w** [comment]
Could you try `pip install hf-doc-builder`? We use [this](https://github.com/huggingface/doc-builder) for building docs. Although, I don't think you need to build docs locally (as it's automatically built by our CI workflows), so just pushing the relevant quality fixes with the correct ruff version is required (because, right now, it's over 3000+ changes now with a different version). Let me know if you're facing any problems, and I'd be happy to push from my end :)

**Comment by ahnjj** [comment]
> so just pushing the relevant quality fixes with the correct ruff version is required (because, right now, it's over 3000+ changes now with a different version). Let me know if you're facing any problems, and I'd be happy to push from my end :)

Thank you so much with your help! 
I made adjustment ( with 0.1.5 version ruff) 
Please let me know if anything left for merge! : >

**Comment by a-r-r-o-w** [comment]
@stevhliu Good to merge? All tests are passing now

**Pull request merged**